service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseController.java

```diff
@@ -85,6 +95,39 @@ private void run2982(CaseDetails caseDetails) {
         caseDetails.getData().put("additionalApplicationsBundle", fixedAdditionalApplicationsBundle);
     }
 
+    private void run2961(CaseDetails caseDetails) {
+        CaseData caseData = getCaseData(caseDetails);
+
+        if (isMigratableState(caseData.getState()) && !containsNoCFields(caseData)) {
+            if (caseData.getRespondents1().size() > 10) {
```

Comment:
If we are going to error out on these cases then you probs want to add it to the filter? That way we won't get any preventable errors in the logs

Reply:
I ran a query after this comment to see how many cases we have with responents.size() > 10. Results shown we do not have any so this we should not get any. Shall we just remove?

Reply:
i think in theory we should migrated such case and have possibility to at least having NoC working for first 10 respondents (we can log warn). But if you checked that it is not a case that is fine

Reply:
I would rather thow exception when isMigratableState(caseData.getState()) && !containsNoCFields(caseData) is not met as this apparently is an error situation as these conditions are check in migration tool